src/styles/utils/_mixins.scss

```diff
@@ -0,0 +1,23 @@
+@mixin squere($size) {
+  width: $size;
+  height: $size;
+}
+
+@mixin flex-center-center() {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+@mixin flex-between-center() {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+@mixin font($size: 14px, $color:$basic-color, $weight: 400, $lh: 1) {
+  font-size: $size;
+  color: $color;
+  font-weight: $weight;
+  line-height: $lh;
```

Comment:
Your code is so good, really))
You shouldn't use variable when you need to use that value just once ($font-size / $font-weight).

Reply:
Thanks) This mixin is general for all font styles, so I like it. Because I can write all font styles in one line)